src/ui/actions/session.ts

```diff
@@ -14,7 +14,7 @@ import LogRocket from "ui/utils/logrocket";
 import { registerRecording, sendTelemetryEvent, trackEvent } from "ui/utils/telemetry";
 import { extractGraphQLError } from "ui/utils/apolloClient";
 
-import { ExpectedError, UnexpectedError } from "ui/state/app";
+import type { ExpectedError, UnexpectedError } from "ui/state/app";
```

Comment:
Didn't know about this!

Reply:
Yeah, TS will let you mix them, like `import {realVariable, SomeType} from "wherever"`, but you can also clearly specify "this line is _only_ type imports" too. Probably are some arguments why that's mostly better, although it can also lead to multiple lines to import types and values separately from one file.